src/sas/qtgui/Utilities/GuiUtils.py

```diff
@@ -1266,6 +1266,10 @@ def jdefault(o):
             content = o.__dict__.copy()
             return add_type(content, type(o))
 
+        if np.isscalar(o):
+            # scalar quantity - no default necessary
+            return None
```

Comment:
`jdefault` should return the serialized version of the value not `None`.

The docs for `np.isscalar` suggest using `isinstance(o, numbers.Number)` if you want to restrict your test to numeric values (note that 'np.isscalar("str")' returns True):
```
import numbers
if isinstance(o, numbers.Number):
    return float(o)
```
However, this will convert all values (int16, int32, int64, float32, float64) to float when they are reloaded.

To preserve int-ness you can use:
```
import numbers
if isinstance(o, numbers.Integral):
    return int(o)
if isinstance(o, numbers.Real):
    return float(o)
```

Note that tuples are treated as lists by json.dump and np.float is deprecated so the first clause should only be checking for sets.

You might be able to dump/load tuples with a specialized JSONEncoder if you care.